app/actions/CustomActionBuilders.scala

```diff
@@ -63,18 +69,18 @@ class CustomActionBuilders(
 
   val PrivateAction = new PrivateActionBuilder(addToken, checkToken, csrfConfig, cc.parsers.defaultBodyParser, cc.executionContext)
 
-  val AuthenticatedAction = PrivateAction andThen authenticated()
+  val AuthenticatedAction = (identityClientId: (String, String)) => PrivateAction andThen authenticated(chooseRegister(identityClientId))
```

Comment:
Could this use a default param, e.g. `membersIdentityClientId`?

Reply:
@Ap0c I tried that but I couldn't figure out how to set a default parameter for a `val`. The best I could do was make it an `Option` then do a `getOrElse` but that still meant passing `None` to it every time which didn't seem any better. Plus I quite like that you have to explicitly pass the clientId, as it forces the developer to think about what they are doing.